components/ProgressCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import '../scss/progressCard.scss'
+
+type Props = {
+  progressCount: number
+}
+
+const ProgressCard: React.FC<Props> = ({ progressCount }) => {
+  return (
+    <div className="progress-card__container card shadow-sm mt-3">
+      <div className="card-body">
+        <div className="progress-card__progress-bar">
+          <span className="font-weight-bold">{progressCount}%</span>
+        </div>
+        <h4 className="progress-card__title mt-3">
```

Comment:
For HTML Semantic you should not add a `H4` if you don't have H1...H3. Please Change for a H1 that you will style it as a h4

Reply:
Thanks learned something. Fixed 👍